spec/features/admin/index/index_spec.rb

```diff
@@ -3,13 +3,29 @@
 RSpec.describe "Admin dashboard" do
   before(:each) do
     @customers = create_list(:customer, 5)
+    @merchant = create(:merchant)
 
-    @customer1_invoice_list = create_list(:invoice, 1, customer: @customers[0])
-    @customer2_invoice_list = create_list(:invoice, 3, customer: @customers[1])
-    @customer3_invoice_list = create_list(:invoice, 5, customer: @customers[2])
+    @item_list1 = create_list(:item, 3, merchant: @merchant, unit_price: 1000)
+    @item_list2 = create_list(:item, 3, merchant: @merchant, unit_price: 1000)
+    @item_list3 = create_list(:item, 3, merchant: @merchant, unit_price: 1000)
+
+
+    @customer1_invoice_list = create_list(:invoice, 1, customer: @customers[0], created_at: "2020-01-01 00:00:00")
+    @customer2_invoice_list = create_list(:invoice, 3, customer: @customers[1], created_at: "2022-01-01 00:00:00")
+    @customer3_invoice_list = create_list(:invoice, 5, customer: @customers[2], created_at: "2021-01-01 00:00:00")
```

Comment:
You can manually set the `created_at` column for any factorybot object using the syntax above